app/frontend/utils/index.ts

```diff
@@ -1,17 +1,18 @@
 import * as humps from "humps"
 import * as R from "ramda"
 
-const SUBMISSION_DATA_REGEX = /^formSubmissionData/
+const SUBMISSION_DATA_PREFIX = "formSubmissionData"
+const SECTION_PREFIX = "section"
```

Comment:
`section` as the prefix will lead to issues if we have any other model property starting with `section`. This is very brittle as `section` is a generic name that could be used in other places and lead to unintentional bugs. We can add in for now, but I highly suggest looking for a work around for this

Reply:
good point